packages/acceptance-tests/pkg-tests-specs/sources/commands/publish.test.ts

```diff
@@ -23,7 +23,7 @@ describe(`npmPublishUtils.getGitHead`, () =>   {
 
     await run(`npm`, `publish`, {
       env: {
-        YARN_NPM_AUTH_TOKEN: `686159dc-64b3-413e-a244-2de2b8d1c36f`,
+        YARN_NPM_AUTH_TOKEN: `316158de-64b3-413e-a244-2de2b8d1c80f`,
```

Comment:
I've had to switch the token for those existing tests here because that token is associated with the user that has `requiresOtp: true` and those tests don't really care about this. They have started to fail after my changes though - so I've switched the user used here to the one with `requiresOtp: false`